setup.py

```diff
@@ -17,21 +14,6 @@
     sys.stderr.write("Warning: Could not open '%s' due %s\n" % ("README.md", error))
 
 
-class CustomInstallCommand(install):
-    def run(self):
-        install.run(self)
-
-
-class CustomDevelopCommand(develop):
-    def run(self):
-        develop.run(self)
-
-
-class CustomEggInfoCommand(egg_info):
-    def run(self):
-        egg_info.run(self)
-
-
```

Comment:
These commands are just not used (I mean, they does not differ from default ones)